### ISSUE TITLE
Import of CSV contacts

### ISSUE THREAD

**Issue by Nikoshi** [DaleFarnsworth-DMR/editcp#10]
The importation of csv files (contacts) would be a handy feature.

**Comment by DaleFarnsworth** [comment]
Agreed.  There is a workaround today, but it's not pretty.  You have to look where editcp downloads its csv file (It appears on the download popup) and then replace that downloaded csv with your own version.  The next time you upload contacts it will use your csv file (provided you uncheck the box that enables a new download by editcp.

It is on my short term (weeks, not years) to-do list.  Thanks.
